TTS/tts/configs/vits_config.py

```diff
@@ -130,13 +131,13 @@ class VitsConfig(BaseTTSConfig):
     add_blank: bool = True
 
     # testing
-    test_sentences: List[str] = field(
+    test_sentences: List[List] = field(
         default_factory=lambda: [
-            "It took me quite a long time to develop a voice, and now that I have it I'm not going to be silent.",
```

Comment:
Because from now on the sentences for testing during training must inform the text, speaker_name, style_wav, language_name. As it was in the previous version it wasn't possible to pass the speaker identity and style_wav, we got a style_wav and speaker embedding randomly. But language is not possible to do this so I proposed this solution now the user must provide for each sentence a list with [text, speaker_name, style_wav, language_name]. However, a list in the formats [text] or [text, speaker_name] or [text, speaker_name, style_wav] is accepted to avoid the user needing to inform items not necessary for the training of the model in question.